letta/server/server.py

```diff
@@ -2125,18 +2115,13 @@ def retry_agent_message(self, agent_id: str) -> List[Message]:
 
     def get_default_user(self) -> User:
 
-        from letta.constants import (
-            DEFAULT_ORG_ID,
-            DEFAULT_ORG_NAME,
-            DEFAULT_USER_ID,
-            DEFAULT_USER_NAME,
-        )
+        from letta.constants import DEFAULT_ORG_ID, DEFAULT_USER_ID, DEFAULT_USER_NAME
```

Comment:
once you have a user model, I'd create a similar `User.default(db_session)` to the Org version and move all that default logic to the model.